backend/side_stacker/game/models.py

```diff
@@ -17,37 +18,47 @@ class Game(models.Model):
     board = models.JSONField(default=initial_board)
     current_player = models.CharField(max_length=1, default='X')
     winner_player = models.CharField(max_length=1, null=True)
+    players = models.IntegerField(default=2)
 
     def execute_movement(self, movement, player):
         row_num = movement['row']
         direction = movement['direction']
         row = self.board[row_num]
         if self.current_player == player:
-            empty_index = None
-            if direction == 'L':
-                i = 0
-                while i < len(row):
-                    piece = row[i]
-                    if piece == 0:
-                        empty_index = i
-                        break
-                    i += 1
-            else:
-                i = len(row) - 1
-                while i >= 0:
-                    piece = row[i]
-                    if piece == 0:
-                        empty_index = i
-                        break
-                    i -= 1
+            empty_index = self.check_if_empty_index(direction, row)
 
             if empty_index is not None:
-                self.place_movement(empty_index, row, row_num)
+                is_winner = self.place_movement(empty_index, row, row_num)
+                # if we are running a 1-player agains bot
+                if not is_winner and self.players == 1:
```

Comment:
I'm testing here if there is a winner after the last user movement, if not and we are playing against the bot we let the bot plays